jest.config.js

```diff
@@ -30,4 +30,5 @@ module.exports = {
     require.resolve("jest-watch-typeahead/filename"),
     require.resolve("jest-watch-typeahead/testname"),
   ],
+  reporters: ["default"],
```

Comment:
where as jest, you can explicitly set the reporter https://jestjs.io/docs/configuration#reporters-arraymodulename--modulename-options